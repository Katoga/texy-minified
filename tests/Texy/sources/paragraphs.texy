.[myclass]
toto je odstavec
 s vice radky


toto je odstavec .[myclass]
 s vice radky


toto je odstavec
 s vice radky .[myclass]


toto je odstavec
 s vice radky
.[myclass]


Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Curabitur turpis enim, placerat tincidunt, tincidunt ac, fringilla et, mauris. .(test){text-align:right}
 A následují řádky
 které Texy! oddělí elementem `<br> &nbsp;`


**Odstavec zarovnaný *doprava.** Lorem* ipsum dolor sit amet, consectetuer adipiscing elit. Curabitur turpis enim, placerat tincidunt, tincidunt ac, fringilla et, mauris. .>


**Odstavec psaný modrou "barvou.** Curabitur":link quam justo, hendrerit a, mattis ut, ultrices at, dui. In mollis. Ut pellentesque erat vehicula nunc. Ut ultricies. Nulla nunc velit, consequat vel, laoreet in, blandit et, eros. .{color:blue}
