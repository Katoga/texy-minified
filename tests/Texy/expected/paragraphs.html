<p class="myclass">toto je odstavec<br />
s vice radky</p>

<p class="myclass">toto je odstavec<br />
s vice radky</p>

<p class="myclass">toto je odstavec<br />
s vice radky</p>

<p class="myclass">toto je odstavec<br />
s vice radky</p>

<p title="test" style="text-align:right">Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Curabitur turpis enim, placerat tincidunt, tincidunt ac, fringilla et, mauris.<br
/>
A následují řádky<br />
které Texy! oddělí elementem <code>&lt;br&gt; &amp;nbsp;</code></p>

<p style="text-align:right"><strong>Odstavec zarovnaný <em>doprava.</em></strong><em> Lorem</em> ipsum dolor sit amet, consectetuer adipiscing elit. Curabitur turpis enim,
placerat tincidunt, tincidunt ac, fringilla et, mauris.</p>

<p style="color:blue"><strong>Odstavec psaný modrou <a href="xxx/link">barvou.</a></strong><a href="xxx/link"> Curabitur</a> quam justo, hendrerit a, mattis ut, ultrices at, dui.
In mollis. Ut pellentesque erat vehicula nunc. Ut ultricies. Nulla nunc velit, consequat vel, laoreet in, blandit et, eros.</p>
